exercises/practice/sgf-parsing/.meta/example.ex

```diff
@@ -134,7 +134,7 @@ defmodule SgfParsing do
       with {:ok, result, rest} <- some(parser).(input) do
         {:ok, result, rest}
       else
-        {:error, _err, ^input} -> {:ok, [], input}
+        {:error, _err, rest} when rest == input -> {:ok, [], input}
```

Comment:
This one seems like a bug to be honest, I'll try to reproduce on something simpler and check.

Reply:
It was fixed in 1.17.1: https://github.com/elixir-lang/elixir/releases/tag/v1.17.1

> [Kernel] Fix bug when using pinned variables inside with's else patterns

Since this is just in an example file, I'm fine completely ignoring this issue (meaning: doing the change you're suggesting to the example file and that's it)